tests/components/technove/snapshots/test_sensor.ambr

```diff
@@ -294,6 +294,7 @@
     'area_id': None,
     'capabilities': dict({
       'options': list([
+        'unknown',
```

Comment:
So instead of adding unknown as an option, can we return None?

Reply:
That's cool, I didn't think of doing it.
This should be alright now.